x/ethbridge/alias.go

```diff
@@ -47,9 +47,9 @@ var (
 )
 
 type (
-	Keeper                       = keeper.Keeper
-	EthBridgeClaim               = types.EthBridgeClaim //nolint:golint
-	OracleClaimContent           = types.OracleClaimContent
+	Keeper         = keeper.Keeper
+	EthBridgeClaim = types.EthBridgeClaim //nolint:golint
+	// OracleClaimContent           = types.OracleClaimContent
```

Comment:
If this isn't used anymore, let's remove.

Reply:
accepted.